pkg/metrics/metrics.go

```diff
@@ -231,6 +245,34 @@ func (m *Metrics) WriteAnchorTime(value time.Duration) {
 	logger.Infof("WriteAnchor time: %s", value)
 }
 
+// WriteAnchorBuildCredentialTime records the time it takes to build credential inside write anchor.
+func (m *Metrics) WriteAnchorBuildCredentialTime(value time.Duration) {
+	m.anchorWriteBuildCredTime.Observe(value.Seconds())
+
+	logger.Infof("WriteAnchor build credential time: %s", value)
```

Comment:
I would make this a debug log. We already have an Info for the whole WriteAnchor time (maybe that should also be a debug).

Reply:
done